internal/provider/resource_environment.go

```diff
@@ -223,6 +223,34 @@ func resourceEnvironment() *schema.Resource {
 					},
 				},
 			},
+			"repositories": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"name": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"database_type": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"allow_provisioning": {
```

Comment:
Again, when would we not allow provisioning?

Reply:
I am not sure i have not seen this as false, this attribute was available in the response so I have just put it output, if we feel it is not required we can remove it also

Reply:
At a minimum, we should default to "true" and set it as not required. 

Let's try to dig into that though. The simpler we can make the configuration, the better.